_posts/2020-04-27-release-2.4.0.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Release v2.4.0
+tags: ["v2.4.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 27/04 2020 the new version 2.4.0 is out
+<!--more-->
+
+## Standardization Improvements
+
+- [#1187]({{ site.github.issues_url }}/1187) If input data does not adhere to schema (complex type to primitive type) it's detected and either new error type is added to `errCol` of the defective row (default) or processing stopped with exception altogether (in case a new switch `--strict-schema-check` is set to `true`)
+
+    Parquet, and and any other future format without enforced schema, now behaves similarly as XML or others on a missing column - it's considered NULL
+
+- [#1285]({{ site.github.issues_url }}/1285) Validate source and raw checkpoints are present in **Standardization_
+
+## Standardization & Conformance Improvements
```

Comment:
Spark Jobs improvements?

Also I would group standardization, this and HyperConformance. They are the same module and each has one change.

Reply:
SparkJobs is the module name. But everywhere in the documentation we refer to each phase as **Standardization** and **Conformance**. And HyperConformance is another logical usage of the module.